api/src/database/migrations/20240716A-update-files-date-fields.ts

```diff
@@ -0,0 +1,23 @@
+import type { Knex } from 'knex';
+
+export async function up(knex: Knex): Promise<void> {
+	await knex.schema.alterTable('directus_files', (table) => {
+		table.renameColumn('uploaded_on', 'created_on');
+	});
+
+	await knex.schema.alterTable('directus_files', (table) => {
+		table.timestamp('uploaded_on');
+	});
+
+	await knex.raw('UPDATE directus_files SET uploaded_on = created_on');
```

Comment:
We should use `knex(table).update()` for this

Reply:
Welp, I tried out the GitHub PR integration and erroneously approved instead of "Request changes". I'll handle the above change and we're good to go 🙂